lib/new_relic/agent/configuration/default_source.rb

```diff
@@ -1692,14 +1691,14 @@ def self.enforce_fallback(allowed_values: nil, fallback: nil)
           :type => Integer,
           :allowed_from_server => false,
           :external => :infinite_tracing,
-          :description => 'Sets the maximum number of span events to buffer when streaming to the trace observer.'
+          :description => 'Sets the maximum number of span events to buffer when streaming to the Trace Observer.'
         },
         :'span_events.max_samples_stored' => {
           :default => 2000,
           :public => true,
           :type => Integer,
           :allowed_from_server => true,
-          :description => 'Defines the maximum number of span events reported from a single harvest. Any Integer between `1` and `10000` is valid.'
+          :description => 'Defines the maximum number of span events reported from a single harvest. Any Integer between `1` and `10000` is valid. Increasing this value may impact memory usage'
```

Comment:
I'm not sure we really need to add this here. It hasn't really been an issue people have asking about in the past and this is generally true for a lot of settings in the agent. So calling out this one config in particular for it doesn't make much sense to me.

Reply:
I've reverted this change. 
https://github.com/newrelic/newrelic-ruby-agent/commit/5476256550f0b182eb525ff29d93e7bddc0ffcaf